Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -7924,7 +7924,7 @@ class HasOnBoundedSide_3 {
                   const Kernel::Point_3&p); 
 
   /*!
-    return true iff the line segment `ab` is inside the union of the
+    returns true iff the line segment `ab` is inside the union of the
```

Comment:
I have modified a bunch of those typos in https://github.com/CGAL/cgal/pull/2315/commits/ba2e820ff0abc7675f64ea0fa401a3edb0e2ca71.